frontend/src/config/i18n/de/baseLayerForm.json

```diff
@@ -4,8 +4,11 @@
   "selectImage": "Hintergrund wählen",
   "rotation_field": "Rotation (Grad)",
   "scale": "Skalierung (pixel/meter)",
+  "scale_invalid": "Die Skalierung muss eine Zahl >= 0 sein!",
```

Comment:
Consider changing >= to greater or equal than.

Reply:
I've now fixed both issues.